app/views/welcome/_user_feed.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="row">
```

Comment:
But it doesn't look like a feed though. I think it would be better to rename it to something like `_guest.html.erb` or `_welcome.html.erb` to make things clearer.